gateway/utils/handlers/config_crud.go

```diff
@@ -560,6 +559,50 @@ func HandleInspectCollectionSchema(adminMan *admin.Manager, modules *modules.Mod
 	}
 }
 
+// HandleInspectUntrackedCollectionSchema gets the schema for particular collection & removed the collection from the database collection schema in config
+func HandleInspectUntrackedCollectionSchema(adminMan *admin.Manager, modules *modules.Modules, syncman *syncman.Manager) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+
+		// Get the JWT token from header
+		token := utils.GetTokenFromHeader(r)
+		defer utils.CloseTheCloser(r.Body)
+
+		// Check if the request is authorised
+		if err := adminMan.IsTokenValid(token); err != nil {
+			_ = utils.SendErrorResponse(w, http.StatusUnauthorized, err.Error())
+			return
+		}
+
+		// Create a context of execution
+		ctx, cancel := context.WithTimeout(r.Context(), 60*time.Second)
+		defer cancel()
+
+		vars := mux.Vars(r)
+		dbAlias := vars["dbAlias"]
+		col := vars["col"]
+		projectID := vars["project"]
+		logicalDBName, err := syncman.GetLogicalDatabaseName(ctx, projectID, dbAlias)
+		if err != nil {
+			_ = utils.SendErrorResponse(w, http.StatusInternalServerError, err.Error())
+			return
+		}
+		schema := modules.Schema()
+		s, err := schema.SchemaInspection(ctx, dbAlias, logicalDBName, col)
```

Comment:
Why are we doing schema inspection here? I don't think so there is any need to inspect schema to untrack a table